### DISCUSSION TITLE
Nametag of waypoint does not disappear in some situation.

### DISCUSSION THREAD

**Discussion by roo-ham** [voxeltycoon/issues#688] [Troubleshooting] [answered]
As a title, nametag of waypoint does not disappear in some situation.

First, remove waypoint that is path of transport line(s).
Second, after removing waypoint, remove path to waypoint.
So gray-colored nametag remains.

It seems developers made nametag-remain-flag for upgrading station, but they couldn't think condition which player remove path after removing station.

Build version
0.85.1 stable

Platform
Windows 10

Reproducibility
Always in same condition

**Comment by victorleblais** [comment, marked as answer]
Nametag of stations and waypoints remain as long as a vehicle has it in its orders.

Remove the order from your vehicles and the tag will disappear instantly.

**Reply by singetail246** [reply]
> 
> 
> Nametag of stations and waypoints remain as long as a vehicle has it in its orders.
> 
> Remove the order from your vehicles and the tag will disappear instantly.

There is a bug with this though. I have about three or four grayed-out stations with no vehicles attached to them and they won't disappear.

**Reply by victorleblais** [reply]
You probably have lines that still exist that use these stops.

Lines do not get removed when there's no vehicles on them (that was a changed asked by several players)
